File: tests/fixtures/simple.js
// Simple JavaScript test program for debugging
function greet(name) {
    const message = `Hello, ${name}!`;
    console.log(message);
    return message;
}

function add(a, b) {
    const result = a + b;
    return result;
}

const name = "World";
const greeting = greet(name);
const sum = add(2, 3);
console.log(`Sum: ${sum}`);

File: tests/fixtures/Simple.java
// Simple Java test program for debugging
public class Simple {
    public static String greet(String name) {
        String message = "Hello, " + name + "!";
        System.out.println(message);
        return message;
    }

    public static int add(int a, int b) {
        int result = a + b;
        return result;
    }

    public static void main(String[] args) {
        String name = "World";
        String greeting = greet(name);
        int sum = add(2, 3);
        System.out.println("Sum: " + sum);
    }
}
